src/amo/components/LandingPage/index.js

```diff
@@ -37,61 +37,88 @@ const ICON_MAP = {
 
 export class LandingPageBase extends React.Component {
   static propTypes = {
-    apiAddonType: PropTypes.func.isRequired,
-    contentForType: PropTypes.func,
+    addonTypeOfResults: PropTypes.string,
     dispatch: PropTypes.func.isRequired,
+    errorHandler: PropTypes.object.isRequired,
     featuredAddons: PropTypes.array,
     highlyRatedAddons: PropTypes.array,
+    loading: PropTypes.boolean,
     popularAddons: PropTypes.array,
     i18n: PropTypes.object.isRequired,
     params: PropTypes.objectOf({
       visibleAddonType: PropTypes.string.isRequired,
     }).isRequired,
+    resultsLoaded: PropTypes.bool.isRequired,
```

Comment:
This could probably be computed from `featuredAddons.length || highlyRatedAddons.length || popularAddons.length` (with a helper function I guess?) rather than needing its own prop.